pollard_test.go

```diff
@@ -1339,3 +1340,148 @@ func compareNodeMap(mapA, mapB map[miniHash]*polNode) error {
 
 	return fmt.Errorf(str)
 }
+
+func TestCachedNodesAfterDelete(t *testing.T) {
+	// Define the test cases as a table.
+	tests := []struct {
+		name            string
+		numAdds         uint64
+		numDels         int
```

Comment:
Why do we have a numDels here? The test can just call `len(test.delIndices)` to get the length.